lib/galaxy/jobs/runners/util/job_script/MEMORY_STATEMENT.sh

```diff
@@ -0,0 +1,4 @@
+if [ -n "$SLURM_JOB_ID" ]; then
+    GALAXY_MEMORY_MB=`scontrol -do show job "$SLURM_JOB_ID" | sed 's/.*\( \|^\)Mem=\([0-9][0-9]*\)\( \|$\).*/\2/p;d'`
```

Comment:
I thought you were going to go out of your way to hide problems if this didn't work automatically?

Reply:
I was working under the false assumption that only the tool's stderr counts anymore. Perhaps that should be the case (in a separate PR)?